src/main/java/Core/Message.java

```diff
@@ -1,11 +1,13 @@
+package Core;
+
 public class Message {
 
-    public static String clientConnected(String username) {
-        return username + " connected.";
+    public static String clientConnected() {
+        return " connected.";
     }
 
-    public static String clientSays(String username) {
-        return "Message from " + username + ": ";
+    public static String clientSays() {
+        return "Core.Message from " + ": ";
```

Comment:
Accidental string concatenation here 😄 
Also, why the need to prefix with `Core.`?